Specification/5._Application_level_conventions.md

```diff
@@ -8,33 +8,31 @@ permalink: /Specification/5._Application_level_conventions/
 
 ### Message type ID
 
-As specified in the CAN bus transport layer specification, the valid range for message type ID is from 0 to 65535,
-inclusive.
+The valid range for *message type ID* is from 0 to 65535, inclusive (specified in [CAN bus transport layer](/Specification/4._CAN_bus_transport_layer)).
 The range is segmented as follows:
 
 ID range        | Purpose
 ----------------|--------------------------------------------------------------
-[0, 20000)      | Standard message types
-[20000, 21000)  | Vendor-specific message types
-[21000, 65536)  | Reserved for future use
+[0, 20000]      | Standard message types
+[20000, 21000]  | Vendor-specific message types
+[21000, 65536]  | Reserved for future use
 
 ### Service type ID
 
-As specified in the CAN bus transport layer specification, the valid range for service type ID is from 0 to 255,
-inclusive.
+The valid range for *service type ID* is from 0 to 255, inclusive (specified in [CAN bus transport layer](/Specification/4._CAN_bus_transport_layer)).
```

Comment:
Seems like the word "specification" is missing from the link here as well? Also below. Or maybe I'm just reading this wrong.